docs/content/developers/building-contributing.md

```diff
@@ -48,6 +50,32 @@ sudo rm /usr/local/bin/ddev
 brew link --force ddev
 ```
 
+### Linux
+
+Start by removing your current binary, for Debian flavor systems use this:
+
+```
+sudo apt remove ddev
+```
+
+Next, unzip the binary you downloaded, make it executable, and move it to a folder in your path. Check with `echo $PATH`:
```

Comment:
Not every distro has `~/.local/bin` in `$PATH`, so you can add an instruction how to do that (e.g. `export PATH="$HOME/.local/bin:$PATH"` to `~/.bashrc`).

Reply:
Great suggestion, I tried to update the text with that tip.